main.go

```diff
@@ -23,6 +24,14 @@ func runTest(name string, client *http.Client, config config.Config) {
 		} else {
 			fmt.Printf("SMS Notification testing completed succesfully")
 		}
+	case "discord":
+		textErr := discord.SendMessage(config.SKU, config.DiscordConfig, client)
+		if textErr != nil {
+			fmt.Printf("Error testing Discord notification exiting...\n")
+			os.Exit(1)
+		} else {
+			fmt.Printf("Discord Notification testing completed succesfully")
+		}
 	default:
 
 	}
```

Comment:
Need to update the logic below this so that it doesn't do an os.Exit after the first test is run for example if a user wants to run with both -sms -discord -test we should allow them to.